src/App.tsx

```diff
@@ -1,18 +1,133 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import './App.scss';
 
+import todos from './api/todos';
 import users from './api/users';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+const preparedTodo: Todo[] = todos.map(task => {
+  return {
+    ...task,
+    user: users.find(user => (user.id === task.userId)) || null,
+  };
+});
 
 const App: React.FC = () => {
+  const [todoList, setTodos] = useState([...preparedTodo]);
+  const [user, setUser] = useState('');
+  const [title, setTitle] = useState('');
+  const [isValideTitle, setIsValideTitle] = useState(false);
+  const [isValideUser, setIsValideUser] = useState(false);
+
+  const selectionCall = 'Chose Your Fighter';
+
+  const addNewTodo = () => {
+    const newTodo: Todo = {
+      userId: 1,
+      id: todoList.length + 1,
+      title,
+      user: users.find(person => person.name === user) || null,
+      completed: false,
+    };
+
+    setTodos([...todoList, newTodo]);
+  };
+
+  const submitAction = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (title === '') {
```

Comment:
```suggestion
    if (title) {
```
стоит также предусмотреть случай когда будет внесена строка только с пробельными символами